assets/js/script.js

```diff
@@ -1,54 +1,111 @@
 var searchInputText = "Hello";
 var searchInputElement = $("#search");
-var wordDefinitionArea = $(".definition");
-var wordPronunciationArea = $("#audio");
+var wordDefinitionArea = $("#definition");
+var wordPronunciationArea = $("#pronunciation");
 var wordExamples = $("#examples");
-var historyContainer = $(".previous-searches"); 
-var searchButton = $("#search-button"); 
+var historyContainer = $(".previous-searches");
+var searchButton = $("#search-button");
 var history = [];
 var favourites = [];
 
-
-const speechURL = "https://voicerss-text-to-speech.p.rapidapi.com/?key=171ec3cab6f247b4b6e7f596d9171ae7&src=" + searchInputText + "&hl=en-us&r=0&c=mp3&f=8khz_8bit_mono";
+const speechURL =
+  "https://voicerss-text-to-speech.p.rapidapi.com/?key=171ec3cab6f247b4b6e7f596d9171ae7&src=" +
+  searchInputText +
+  "&hl=en-us&r=0&c=mp3&f=8khz_8bit_mono";
 const speechOptions = {
-	method: 'GET',
-	headers: {
-		'X-RapidAPI-Key': '6258e18b25mshd96a594dcf7fcb5p1b1fd6jsn955bae6d8f8b',
-		'X-RapidAPI-Host': 'voicerss-text-to-speech.p.rapidapi.com'
-	}
+  method: "GET",
+  headers: {
+    "X-RapidAPI-Key": "6258e18b25mshd96a594dcf7fcb5p1b1fd6jsn955bae6d8f8b",
+    "X-RapidAPI-Host": "voicerss-text-to-speech.p.rapidapi.com",
+  },
 };
 
 const wordsURL = "https://wordsapiv1.p.rapidapi.com/words/" + searchInputText + "/definitions";
 const wordsOptions = {
-	method: 'GET',
-	headers: {
-		'X-RapidAPI-Key': '6258e18b25mshd96a594dcf7fcb5p1b1fd6jsn955bae6d8f8b',
-		'X-RapidAPI-Host': 'wordsapiv1.p.rapidapi.com'
-	}
+  method: "GET",
+  headers: {
+    "X-RapidAPI-Key": "6258e18b25mshd96a594dcf7fcb5p1b1fd6jsn955bae6d8f8b",
+    "X-RapidAPI-Host": "wordsapiv1.p.rapidapi.com",
+  },
 };
 
 //$(".search-button").on("click", function () {
-    
-  //});
 
+//});
+
+function onsearch() {
+  fetch(wordsURL, wordsOptions)
+    .then(function (response) {
+      return response.json();
+    })
+    .then(function (data) {
+      console.log("words data: ", data);
+    });
+
+  fetch(speechURL, speechOptions).then(function (response) {
+    console.log(response);
+  });
+}
+
+// onsearch();
```

Comment:
I just commented out where you calling onsearch(). This stopped the console.logs within your function getting in the way. This was/is just for your (@HillaryM17) testing purposes I hope. :)

I've actually left a similar redundant test call to my function renderWord() as well. (However there are no console.logs within this function, so should not interfere fingers crossed).